website/docs/docs/deploy/continuous-integration.md

```diff
@@ -27,6 +27,27 @@ When the CI run completes, you can view the run status directly from within the
 
 dbt Cloud deletes the temporary schema from your <Term id="data-warehouse" /> when you close or merge the pull request. If your project has schema customization using the [generate_schema_name](/docs/build/custom-schemas#how-does-dbt-generate-a-models-schema-name) macro, dbt Cloud might not drop the temporary schema from your data warehouse. For more information, refer to [Troubleshooting](/docs/deploy/ci-jobs#troubleshooting).
 
+## Git providers who support CI
+
+If your git provider has a native dbt Cloud integration, you can set up continuous integration (CI) jobs within dbt Cloud.
+
+| **Git provider** | **Native dbt Cloud integration** |[**Automated CI job**](/docs/deploy/ci-jobs)|**Git import with URL**| **Information**|
+| -----------------| ---------------------------------| -------------------------------------------|-----------------------|---------|
+|[Azure DevOps](/docs/cloud/git/setup-azure) <Lifecycle status="enterprise" />|  ✅|  ✅ |  ✅  | Organizations on the Team Plan can connect to Azure DeveOps via deploy key. Note, you won’t get the automated CI jobs but you can still develop.|
```

Comment:
wondering if we should add a line break so the lifecycle pills are on a 2nd line? do you think the git provider column looks a bit busy?

![Screenshot 2024-12-19 at 14 36 42](https://github.com/user-attachments/assets/8ad4c9bb-c753-469c-82c1-2b69e49d1ec3)


```suggestion
|[Azure DevOps](/docs/cloud/git/setup-azure)<br /> <Lifecycle status="enterprise" />|  ✅|  ✅ |  ✅  | Organizations on the Team Plan can connect to Azure DeveOps via deploy key. Note, you won’t get the automated CI jobs but you can still develop.|
```